pkg/sdkserver/routes.go

```diff
@@ -58,6 +58,11 @@ func (s *server) addRoutes(mux *http.ServeMux) {
 	mux.HandleFunc("POST /confirm/{id}", s.confirm)
 	mux.HandleFunc("POST /prompt/{id}", s.prompt)
 	mux.HandleFunc("POST /prompt-response/{id}", s.promptResponse)
+
+	mux.HandleFunc("POST /credentials", s.listCredentials)
+	mux.HandleFunc("POST /credentials/create", s.createCredential)
+	mux.HandleFunc("POST /credentials/reveal", s.revealCredential)
```

Comment:
Let me know if these routes should change at all or if this is fine. Feels kinda weird to have it this way, but it's also abnormal to do a request body with a GET, so...I just did this.

Reply:
The alternative is to use path and query parameters (e.g. `/credentials/reveal/name?context=my-context` or `/credentials/delete/name?context=my-context` (and make this `DELETE`)).

I leave the ultimate decision up to you. We certainly aren't building a REST API. This is more of RPC stuff, which I think is almost always `POST`.

I'll leave a final review after your response.

Reply:
Cool, I think I will leave it as it is then. All POST is what works best with the way the Go SDK is set up, at least.